recipes/esmpy/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+export ESMFMKFILE=$(find ${PREFIX} -name '*esmf.mk')
```

Comment:
@bekozi this is a little bit fishy. Maybe we can set most of the information in that file as env vars?